shell/pages/auth/setup.vue

```diff
@@ -459,7 +459,9 @@ export default {
   }
 
   .landscape {
-    background-image: url('~shell/assets/images/pl/login-landscape.svg');
+    /* Added by Verrazzano Start */
```

Comment:
Comment out rather than delete the original line and move the start comment line above it.